internal/common/platforms.go

```diff
@@ -0,0 +1,66 @@
+package common
+
+import (
+	"github.com/openshift/assisted-service/models"
+	"github.com/thoas/go-funk"
+)
+
+const (
+	VmwareManufacturer string = "VMware, Inc."
+)
+
+var supportedManufacturers = map[string]models.PlatformType{
+	VmwareManufacturer: models.PlatformTypeVsphere,
+}
+
+func GetAllSupportedPlatforms() *[]models.PlatformType {
+	return &[]models.PlatformType{
+		models.PlatformTypeVsphere,
+		models.PlatformTypeBaremetal,
+	}
+}
+
+func GetPlatformFromVendor(vendor models.SystemVendor) models.PlatformType {
+	return supportedManufacturers[vendor.Manufacturer]
+}
+
+func GetHostSupportedPlatforms(inventory models.Inventory) *[]models.PlatformType {
```

Comment:
Host is a bit confusing, you don't pass a host here. 
```suggestion
func GetSupportedPlatformsFromInventory(inventory models.Inventory) *[]models.PlatformType {
```